tests/combiners_test.py

```diff
@@ -111,9 +111,21 @@ def test_create_compound_combiner(self, metrics, expected_combiner_types):
 
         # Mock budget accountant.
         budget_accountant = mock.Mock()
-        mock_budgets = [
-            f"budget{i}" for i in range(len(expected_combiner_types))
-        ]
+        n_budget_requests = len(expected_combiner_types)
+        if dp_combiners.MeanCombiner in expected_combiner_types:
+            n_budget_requests += 1
+        mock_budgets, expected_budgets = [], []
+        for i, expeced_type in enumerate(expected_combiner_types):
+            if expeced_type == dp_combiners.MeanCombiner:
+                # MeanCombiner requests budgets twice, for Count and for Sum.
```

Comment:
super-nit: `one for Count and one for Sum`

Reply:
thx